components/supervisor/pkg/supervisor/supervisor.go

```diff
@@ -810,6 +810,7 @@ func configureGit(cfg *Config) {
 		{"alias.lg", "log --color --graph --pretty=format:'%Cred%h%Creset -%C(yellow)%d%Creset %s %Cgreen(%cr) %C(bold blue)<%an>%Creset' --abbrev-commit"},
 		{"credential.helper", "/usr/bin/gp credential-helper"},
 		{"safe.directory", "*"},
+		{"core.hooksPath", "/etc/git/hooks"},
```

Comment:
Will this affect hooks in repository?

Reply:
Oh man, great catch! This would probably totally mess up repos with their own hooks and override them. This is very blocking, and I'm hence drafting this PR again, until I figure out a way to do this without causing issues to hook-enabled setups.

Reply:
I've changed the behavior now to inject straight into the cloned repo under `/workspace/` in [`607a509` (#20525)](https://github.com/gitpod-io/gitpod/pull/20525/commits/607a509b416d0641d3de674a20b4490b711939d4). [Here](https://linear.app/gitpod/issue/CLC-1040/tagging-commits-from-gitpod-workspaces#comment-10d6bddf) are some tests I did with popular git hook management tools.